### ISSUE TITLE
ateste

### ISSUE THREAD

**Issue by gustavoluizd16-star** [gustavoluizd16-star/chatbot#1]
import streamlit as st
import pandas as pd
import numpy as np

# Configuração visual do App
st.set_page_config(page_title="MATRIX-LOGIC v3.0", layout="wide")

st.title("🖥️ DASHBOARD ESTRATÉGICO MATRIX (BAC BO)")

# --- SIMULAÇÃO DE DADOS (Aqui entrará sua conexão com o histórico) ---
# No futuro, você pode conectar isso a uma API ou planilha
forca_vermelho = 72 
forca_azul = 28

# --- 1. MONITOR DE FORÇA E VOLUME ---
st.header("1. Monitor de Força e Volume")
col1, col2 = st.columns(2)

with col1:
    st.subheader(f"🔴 FORÇA VERMELHO: {forca_vermelho}%")
    st.progress(forca_vermelho / 100)
    st.caption("STATUS: SURF ATIVO")

with col2:
    st.subheader(f"🔵 FORÇA AZUL: {forca_azul}%")
    st.progress(forca_azul / 100)
    st.caption("STATUS: QUEBRA FRACA")

# --- 2. RANKING DE SOMAS VICIADAS ---
st.subheader("📊 Ranking de Somas Viciadas")
dados_somas = {
    "Soma": ["9R", "10R", "11R"],
    "Frequência": [22, 18, 15],
    "Assertividade": ["92%", "88%", "95%"]
}
st.table(pd.DataFrame(dados_somas))

# --- 3. MATRIZ DE PUXADAS (SINAL EM TEMPO REAL) ---
st.divider()
st.header("🎯 AGENTE DE ANÁLISE: SINAL ATUAL")

# Exemplo de lógica de decisão baseada no seu texto
soma_atual = 9 # Simulação
cor_atual = "Vermelho"

if soma_atual in [9, 10, 11] and cor_atual == "Vermelho":
    st.success("🔥 ENTRADA CONFIRMADA: REPETIR VERMELHO (Mão Pesada)")
    st.metric(label="Confiança", value="95%")
elif soma_atual == 12:
    st.warning("⚠️ SINAL DE ALERTA: INVERTER COR (Quebra de Teto)")
    st.metric(label="Confiança", value="92%")
else:
    st.info("⌛ AGUARDANDO PADRÃO...")

# --- 4. GESTÃO DE BANCA ---
with st.sidebar:
    st.header("💰 Gestão de Capital")
    banca_inicial = st.number_input("Sua Banca (R$)", value=1000)
    meta = banca_inicial * 1.20
    stop_loss = banca_inicial * 0.90
    
    st.write(f"🚀 **Meta (Stop Win):** R$ {meta:.2f}")
    st.write(f"🛑 **Limite (Stop Loss):** R$ {stop_loss:.2f}")
